packages/core/src/constant.ts

```diff
@@ -41,7 +41,7 @@ export const EVENTS = {
   SESSION_EXTENDED: "sessionExtended",
   SESSION_RESTORED: "sessionRestore",
   TIMEOUT_SET: "timeoutSet",
-};
+} as const;
```

Comment:
Though unlikely, this may be a breaking change for consumers if they have any types expecting this to be a `Record<string, string>`.

Reply:
That's correct, and the same applies to users using a session instance as an arbitrary event emitter: the type overloads we are specifying don't allow for non-session specific events. Since none of this is public API (core being an internal module to the public the browser and node packages), I think it is unlikely enough that downstream users are relying on this to make it okay for this to change. In particular, not making this a `const` prevents from getting useful type hints in the `SessionEventEmitter` interface, as `typeof EVENTS.LOGIN` would be `string` rather than `"login"`.